mmv1/third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -125,17 +126,22 @@ func isSubnetworkUnreadyError(err error) (bool, string) {
 	return false, ""
 }
 
+var QuotaRegex = regexp.MustCompile(`Quota exceeded for quota metric '(?P<Metric>.*)' and limit '(?P<Limit>.* per minute)' of service 'compute.googleapis.com'`)
```

Comment:
nit: Given this is used in a single function, let's scope it into that function rather than making it package-scope (which is really global, since we use a single package)

Reply:
I originally had it scoped inside the function but realized that the regex will be compiled every time the function is called so for performance reasons I decided to move it into the package-scope. LMK if you feel that performance is not a concern and I'll be happy to move it into the function.

Reply:
Ehh- probably not a big deal.

Reply:
Sure, will move the regex back into the function

Reply:
Done